browser-test/src/support/admin_questions.ts

```diff
@@ -789,10 +789,11 @@ export class AdminQuestions {
       )
     }
 
-    assert(options)
-    for (let index = 0; index < options.length; index++) {
+    expect(options).toBeTruthy()
+
+    for (let index = 0; index < options!.length; index++) {
       await this.page.click('#add-new-option')
-      await this.fillMultiOptionAnswer(index, options[index])
+      await this.fillMultiOptionAnswer(index, options![index])
```

Comment:
question (non-blocking): what does adding `!` here do?

Reply:
This tells the Typescript compiler to be chill with its not null check, otherwise it complains. The previous `expect` on line 792 will fail if it's null. I changed it because it was causing failures when running.